### PR TITLE
Refator Code to use `!is` instead of `!(<variable> is <type>)`

### LINKED ISSUE #2952: Format the generated functions for oneof type
**Description:**
Need to apply these changes https://github.com/ballerina-platform/module-ballerina-protobuf/pull/176#discussion_r884670782

### PR DESCRIPTION
**Pull request by Ishad-M-I-M**
## Purpose
Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/2952
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec

### CODE REVIEW

File: tooling-tests/src/test/resources/test-src/generated-sources/tool_test_data_type_9/helloWorldWithReservedNames_pb.bal [hunk 1]
```diff
@@ -161,7 +161,7 @@ public type FieldRules record {|
 
 isolated function isValidFieldrules(FieldRules r) returns boolean {
     int typeCount = 0;
-    if !(r?.'enum is ()) {
+    if r?.'enum !is () {
         typeCount += 1;
     }
     if (typeCount > 1) {
```
Comment (dilanSachi) [inline tooling-tests/src/test/resources/test-src/generated-sources/tool_test_data_type_9/helloWorldWithReservedNames_pb.bal]:
```suggestion
    if typeCount > 1 {
```
Shall we do this fix as well?

File: tooling-tests/src/test/resources/test-src/generated-sources/tool_test_data_type_11/oneof_field_service_pb.bal [hunk 2]
```diff
@@ -166,23 +166,23 @@ public type Request1 record {|
...
     }
-    if !(r?.last_name is ()) {
+    if r?.last_name !is () {
         nameCount += 1;
     }
-    if !(r?.'version is ()) {
+    if r?.'version !is () {
         nameCount += 1;
     }
     if (otherCount > 1 || nameCount > 1) {
```
Comment (dilanSachi) [inline tooling-tests/src/test/resources/test-src/generated-sources/tool_test_data_type_11/oneof_field_service_pb.bal]:
```suggestion
    if otherCount > 1 || nameCount > 1 {
```
Here also

File: tooling-tests/src/test/resources/test-src/generated-sources/tool_test_data_type_9/helloWorldWithReservedNames_pb.bal (hunk 1 above)
Comment (Ishad-M-I-M) [reply tooling-tests/src/test/resources/test-src/generated-sources/tool_test_data_type_9/helloWorldWithReservedNames_pb.bal]: Done

File: tooling-tests/src/test/resources/test-src/generated-sources/tool_test_data_type_11/oneof_field_service_pb.bal (hunk 2 above)
Comment (Ishad-M-I-M) [reply tooling-tests/src/test/resources/test-src/generated-sources/tool_test_data_type_11/oneof_field_service_pb.bal]: Done

**Review by dilanSachi — APPROVED** [review]

**Pull request merged**